versioned_docs/version-1.12/self-hosted/administration/configuration.mdx

```diff
@@ -973,8 +973,8 @@ namespace:
 Configures [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) for each namespace to isolate network traffic.
 Disabled by default.
 
-- `ingress`: Ingress list of rules ([NetworkPolicyIngressRule](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#networkpolicyingressrule-v1-networking-k8s-io)) to be applied on every namespace managed by Okteto. (optional).
-- `egress`: Egress list of rules ([NetworkPolicyEgressRule](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#networkpolicyegressrule-v1-networking-k8s-io)) to be applied on every namespace managed by Okteto (optional).
+- `ingress`: Ingress list of rules ([NetworkPolicyIngressRule](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#networkpolicyingressrule-v1-networking-k8s-io)) to be applied on every namespace managed by Okteto. (optional).
```

Comment:
Will 1.25 links stop working when new versions come as 1.22 links did?
is there a permanent link we can use here?

Reply:
That is a great point. I am actually surprised they got rid of the page for the older versions. I'll check if there's a permanent link.

Reply:
There doesn't seem to be  one for 'latest' that could be used permanently. This link might need to be updated to the Kubernetes version each time we change the Kubernetes version with a new Chart release. 

Is that a good approach?

Reply:
Removed the links on all the doc versions, since they are all broken